client/src/components/QuestionsAnswers.jsx

```diff
@@ -9,30 +10,36 @@ export default function QuestionsAnswers({ productID }) {
   const tempId = 40355;
   const [questionList, setQuestionList] = useState([]);
   const [filteredList, setFilteredList] = useState([]);
+  const [show, setShow] = useState(false);
 
-  useEffect(() => {
+  const updateQuestions = () => {
+    console.log('invoked');
     axios.get(`/api/qa/questions?product_id=${tempId}&count=5&page=1`)
       .then((result) => {
         setQuestionList(result.data.results);
       })
       .catch((err) => {
         console.log(err);
       });
+  };
+  useEffect(() => {
+    updateQuestions();
   }, [tempId]);
 
   useEffect(() => {
     setFilteredList(questionList);
```

Comment:
on line 30, it seems that filteredList is the same as the questionList? i thought that there should be some filter function?

Reply:
I don't know if I understood your question correctly but this is just presetting my filtered list and later on it's state will be changed by other functions
and my list will stay the same for returning back to original state.

Reply:
gotcha